foundations/installations/installing_node.md

```diff
@@ -1,12 +1,17 @@
 ### Introduction
 
-Node is a JavaScript interpreter that doesn't run in the browser, there are many tools that require this to be installed. Testing frameworks and bundlers are two of many.
+Node.js is a JavaScript runtime environment, that means that now you can run JavaScript code on the server side also!
+Node uses JavaScript as its scripting language, and achieves high throughput via non-blocking I/O and a single-threaded event loop.
+
+There are many tools that require this to be installed. Testing frameworks and bundlers are two of many.
 
 We're going to install it using `nvm` (Node Version Manager), because it makes it easy to change node versions and upgrade node.  There is another tool called `npm` (Node Package Manager) that you will use later to install the various libraries and tools used in javascript environment.  It can be easy to confuse these two so read carefully!
 
 Node is also very easy to install using nvm, so this should go quickly :)
 
-### Step 0: Prerequisites For Linux
+# Installation in Linux
```

Comment:
The Linux instructions should also be followed by OS X users (we mention OS X in this section, and we assume that all students are on one of the two), why don’t we change this header to say Linux / OS X, and only mention the zsh_profile and zshrc on line 51 instead of breaking Mac OS X out into a separate section?

Reply:
I guess ,it is is more cleaner this way!
using bash equivalent to zsh_profile and zshrc, one will get into permission issues .